diagnostic.md

```diff
@@ -5,74 +5,85 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+Backend is necessary for webpages because it allows for interactions among different users on different devices, and allows data to be saved over time. For example, if two people are using a webpage, they have two different copies and can never interact with each other unless there is backend. Similarly, you would need backend if you want to save your game and come back to the same place later.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+Model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+Controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+We use 'serializers' that act like the View layer.
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+Create, Read, Update, Destroy
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+Model (M)
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+GET (all), GET (single), PATCH, POST, DELETE, PUT
 ```
 
 A user action fires a `GET` request for `/people/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list)
 
 ```md
-// your response here
+- User fires 'GET' request for 'people/1'
+- API sends request to Controller
+- Controller parse user request, sends request to Model
+- Model retrieves people item with index 1 from the peoples table of the people database, sends back data to Controller
+- Controller sends data to View
+- View formats the data, sends it back to Controller
+- Controller sends it back to API
+- API sends back requested data to Browser (client)
 ```
 
 What is the command to generate a new rails-api app?
 
 ```bash
-// your response here
+bin/rails generate
```

Comment:
I think we need a little more here: `rails-api new app_name -T --database=postgresql`